demos/apps/echo-with-auth/README.md

```diff
@@ -0,0 +1,55 @@
+# Istio Kubernetes Demo with simple echo app
+
+**Install Istio on a Kubernetes cluster**
+
+1. Install Istio Manager
+    ```bash
+    kubectl apply -f istio-manager.yaml
+    ```
+
+    Notice in the YAML file, the manager has been configured to use mutual TLS connections between Istio proxies
+    ```yaml
+    data:
+      mesh: |-
+        authPolicy: 1                        # Enables Istio auth
+        mixerAddress: istio-mixer:9091
+        discoveryAddress: istio-manager:8080
+    ```
+1. Install Istio Mixer
+    ```bash
+    kubectl apply -f istio-mixer.yaml
+    ```
+1. Install Istio CA
+    ```bash
+    kubectl apply -f istio-ca.yaml
+    ```
+
+**Deploy a simple echo app with manually injected proxy**
+```bash
+kubectl apply -f <(istioctl kube-inject -f echo-app.yaml --tag 2017-04-08-23.03.08)
+kubectl apply -f <(istioctl kube-inject -f logic-app.yaml --tag 2017-04-08-23.03.08)
+```
+
+This will deploy two pods, each running a simple echo server and client, and
+will create two kubernetes services called "echo" and "logic".
+
```

Comment:
They are backed by the same app, I just named the k8s services differently. It's better to switch to a curl or wrk client and a simple http server.